src/Transaction.purs

```diff
@@ -130,6 +132,12 @@ attachPlutusScripts ps tx@(Transaction { witnessSet: ws }) = do
   updateTxWithWitnesses tx
     =<< convertWitnessesWith ws (Serialization.WitnessSet.setPlutusScripts ps')
 
+attachNativeScript
```

Comment:
Once again docs can be straightforwardly added as in my previous comment

Reply:
This function is internal and the purpose is obvious